examples/_dev/package.json

```diff
@@ -13,7 +13,7 @@
     "next": "^12.2.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "wagmi": "^0.6.4"
+    "wagmi": "workspace:*"
```

Comment:
Wasn't this working before with `preconstruct dev`?

Reply:
This was to fix issues with linking `@wagmi/core`, but since we merged the lockfile fix, we don't need this anymore!